### ISSUE TITLE
EL9: more runtime requirements

### ISSUE THREAD

**Issue by pille** [matteocorti/check_updates#56]
**Describe the bug**

on EL9 (e.g. AlmaLinux 9.0) the following additional runtime requirements are needed:

- `perl-English`
- `perl-Readonly` (fixed in #55)
- `perl-Monitoring-Plugin`
**To Reproduce**

freshly install AlmaLinux 9.0 and check_updates.

**Expected behavior**

plugin works

**System (please complete the following information):**

 - OS: EL9 (AlmaLinux 9)
 - OS Version: 9.0
 - check_updates Version: 2.0.0

**Comment by matteocorti** [comment]
Already fixed with #55 

```
$ rpm -q --requires /home/corti/rpmbuild/RPMS/x86_64/nagios-plugins-check-updates-2.0.2-3.fc36.x86_64.rpm
/usr/bin/perl
nagios-plugins
perl(Carp)
perl(English)
perl(Monitoring::Plugin)
perl(POSIX)
perl(Readonly)
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsZstd) <= 5.4.18-1
```

**Comment by matteocorti** [comment]
Tested on CentOS 9
